### PR TITLE
Add prod run for configuration

### PR DESCRIPTION
**Pull request by mirnawong1**
Clarifies that a job run is needed in order for `access` config to take effect.

Raised by a user via cs. Slack thread for more context:  https://dbt-labs.slack.com/archives/C019QLR6EJE/p1709661008474459

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/collaborate/govern/project-dependencies.md [hunk 1]
```diff
@@ -34,9 +34,10 @@ Refer to the [FAQs](#faqs) for more info.
 In order to add project dependencies and resolve cross-project `ref`, you must:
 - Use dbt v1.6 or higher for **both** the upstream ("producer") project and the downstream ("consumer") project.
 - Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access)
+  - Once defined, re-run a production job in order to apply the change.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/govern/project-dependencies.md]:
```suggestion
  - Once defined, re-run a production job to apply the change.
```

File: website/docs/docs/collaborate/govern/project-dependencies.md [hunk 2]
```diff
@@ -34,9 +34,10 @@ Refer to the [FAQs](#faqs) for more info.
 In order to add project dependencies and resolve cross-project `ref`, you must:
 - Use dbt v1.6 or higher for **both** the upstream ("producer") project and the downstream ("consumer") project.
 - Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access)
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/govern/project-dependencies.md]:
```suggestion
- Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access). Once defined, re-run a production job to apply the change.
```

File: website/docs/docs/collaborate/govern/project-dependencies.md [hunk 3]
```diff
@@ -34,9 +34,10 @@ Refer to the [FAQs](#faqs) for more info.
...
+  - Once defined, re-run a production job to apply the change.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/govern/project-dependencies.md]:
```suggestion
```

File: website/docs/docs/collaborate/govern/model-access.md [hunk 4]
```diff
@@ -59,9 +59,10 @@ Some models are implementation details, meant for reference only within their gr
 
 | Access    | Referenceable by                       |
 |-----------|----------------------------------------|
-| private   | same group                             |
-| protected | same project (or installed as package) |
-| public    | any group, package or project          |
+| private   | Same group                             |
+| protected | Same project (or installed as a package) |
+| public    | Any group, package, or project* |
```
Comment (nghi-ly) [inline website/docs/docs/collaborate/govern/model-access.md]: sugg: 
- short enough to stay in table 
- https://www.merriam-webster.com/dictionary/rerun -- American English tend to _not_ use hyphens for prefixes like British English do

```suggestion
| public    | Any group, package, or project. When defined, rerun a production job to apply the change |
```

File: website/docs/docs/collaborate/govern/model-access.md [hunk 5]
```diff
@@ -59,9 +59,10 @@ Some models are implementation details, meant for reference only within their gr
...
+*Once defined, re-run a production job to apply the change.
```
Comment (nghi-ly) [inline website/docs/docs/collaborate/govern/model-access.md]:
```suggestion
```

File: website/docs/docs/collaborate/govern/project-dependencies.md [hunk 6]
```diff
@@ -33,10 +33,10 @@ Refer to the [FAQs](#faqs) for more info.
 
 In order to add project dependencies and resolve cross-project `ref`, you must:
 - Use dbt v1.6 or higher for **both** the upstream ("producer") project and the downstream ("consumer") project.
-- Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access)
+- Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access). Once defined, re-run a production job to apply the change.
```
Comment (nghi-ly) [inline website/docs/docs/collaborate/govern/project-dependencies.md]:
```suggestion
- Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access). To apply the change, rerun a production job.
```

File: website/docs/reference/resource-configs/access.md [hunk 7]
```diff
@@ -66,16 +66,18 @@ select ...
 
 </VersionBlock>
 
+Note, once you define `access`, re-run a production job to apply the change.
```
Comment (nghi-ly) [inline website/docs/reference/resource-configs/access.md]:
```suggestion
After you define `access`, rerun a production job to apply the change. 
```

File: website/docs/reference/resource-configs/access.md [hunk 8]
```diff
@@ -66,16 +66,18 @@ select ...
...
 Some models (not all) are designed to be referenced through the [ref](/reference/dbt-jinja-functions/ref) function across [groups](/docs/build/groups).
 
 | Access    | Referenceable by              |
 |-----------|-------------------------------|
-| private   | same group                    |
-| protected | same project/package          |
-| public    | any group, package or project |
+| private   | Same group                    |
+| protected | Same project/package          |
+| public    | Any group, package, or project*|
```
Comment (nghi-ly) [inline website/docs/reference/resource-configs/access.md]:
```suggestion
| public    | Any group, package, or project. When defined, rerun a production job to apply the change. |
```

**Review by nghi-ly — COMMENTED** [review]
looks good! left several suggs for ya, minor nits

**Review by nghi-ly — APPROVED** [review]

**Pull request merged**